docs/content/workers/api-worker-system.md

```diff
@@ -31,6 +31,17 @@ Parameters:
 
 Returns: `true`, if the queried SpatialOS entity is checked out on this worker, `false` otherwise.
 
+
+```csharp
+bool HasEntity(EntityId entityId);
+```
+
+Parameters:
+
+  * `EntityId entityId`: The entity ID of the SpatialOS entity you want to check.
+
+Returns: `true`, if the SpatialOS entity is checked out on this worker, `false` otherwise.
```

Comment:
```suggestion
Returns: `true` if the SpatialOS entity is checked out on this worker, `false` otherwise.
```

Reply:
Small (and I know it was already there) but comma not needed. Not a show-stopper. Just a niggle.